card/lib/generators/card.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
```

Comment:
This file just defines a couple of classes that are used by the actual generators.  The main reason *rails g* wasn't working quite right was I wasn't including this file in each generator.